helm/cluster/templates/clusterapi/_helpers_postkubeadmcommands.tpl

```diff
@@ -0,0 +1,16 @@
+{{- define "cluster.internal.kubeadm.postKubeadmCommands" }}
```

Comment:
Helper templates for specifying post-kubeadm commands (commands that run after kubeadm in all nodes) which are common for both control plane (KubeadmControlPlane) and worker nodes (KubeadmConfig).